data.js

```diff
@@ -1265,6 +1265,18 @@ module.exports = [
       'Umbraco'
     ]
   },
+  {
+    name: '@litbelb on Medium',
+    description: 'Gopher, Student, Gamer, very unprofessional writer',
+    url: 'https://medium.litbelb.com',
+    twitter: '@Litbelb',
+    tags: [
+      'Golang',
+      'Frontend',
```

Comment:
Just an small update :) Could you please change Frontend by Front End, to keep consistency between tags.
Thanks a lot!!

Reply:
Got it fixed!